tests/integration/pkg/tests/testdata/sklearn-iris.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Model
+metadata:
+  name: iris
+spec:
+  storageUri: "gs://seldon-models/scv2/samples/rolling/iris/v1"
+  requirements:
+  - sklearn
```

Comment:
:thought_balloon: There's no `memory` specified here as for `tensorflow-tfsimple.yaml`

Reply:
https://github.com/SeldonIO/seldon-core/pull/4895/commits/a7974130467108b25cd667fe969b878a68952438